CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- *Breaking* Changed how backup details are stored in the repository to 
```

Comment:
Will update this possibly in a follow up once the change in behavior is documented